src/intrinsics.cr

```diff
@@ -1,6 +1,6 @@
 lib Intrinsics
   fun debugtrap = "llvm.debugtrap"
-  {% if flag?(:x86_64) %}
+  {% if flag?(:x86_64) || flag(:aarch64) %}
```

Comment:
Would it not make more sense to check the type of `LibC::SizeT` here? It's beyond the scope of this PR because I don't think there's a way because of how aliases are handled in macros right now, but it might be a good idea as we add more 64bit platforms.